browser/admin/src/integrator/WordBook.ts

```diff
@@ -342,7 +342,12 @@ class WordBook {
 		addButton.disabled = true;
 		newWordInput.addEventListener('input', () => {
 			const newWord = newWordInput.value.trim();
-			addButton.disabled = newWord === '';
+
+			const wordExists = this.currWordbookFile.words.some(
```

Comment:
do we want to do it on every keypress? in real time?
maybe `change` event would be better for performance reason - only after field is left it will be triggered

Reply:
Make sense, may be it's fine to not have it on real time.